__tests__/Unit/Components/Tasks/ProgressUpdateCard/ProgressUpdateCardPresentation.test.tsx

```diff
@@ -123,7 +129,76 @@ describe('ProgressUpdateCardPresentation Component', () => {
             onCardClick: mockedOnCardClick,
         };
     });
+
+    it('should rotate the angle icon when expanded', () => {
+        (useRouter as jest.Mock).mockReturnValue({
+            query: { dev: 'false' },
+        });
+        const props = { ...initialProps, isExpanded: true };
+        renderWithRouter(
```

Comment:
Does this compnent need router access??

Reply:
changes are currently behind  feature flag, we need to use `useRouter` to get the query params to conditionally render changes